### PR TITLE
Desktop app v5.8 changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by cwarnermm — APPROVED** [review]

File: source/deploy/desktop-app-changelog.rst [hunk 1]
```diff
@@ -3,12 +3,73 @@ Desktop application changelog
 
 This changelog summarizes updates to Mattermost desktop app releases for `Mattermost <https://mattermost.com>`__.
 
+Release v5.8
+--------------
+
+**Release Date: April 16, 2024**
```
Comment (marianunez) [inline source/deploy/desktop-app-changelog.rst:9]: Should this be May ?

**Review by marianunez — APPROVED** [review]

**Comment by devinbinnie** [comment]
@amyblais Need to add this changes:
https://github.com/mattermost/desktop/pull/3010

File: source/deploy/desktop-app-changelog.rst [hunk 2]
```diff
@@ -3,12 +3,73 @@ Desktop application changelog
...
+
+All Platforms
+^^^^^^^^^^^^^
+
+- Moved the **Settings** window into a new modal.
+
+Architectural Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Major version upgrade of Electron to 29.3.0. Electron is the underlying technology used to build the Desktop App.
```
Comment (devinbinnie) [inline source/deploy/desktop-app-changelog.rst:30]: Just a minor version this time.

**Review by devinbinnie — APPROVED** [review]

File: source/deploy/desktop-app-changelog.rst [hunk 3]
```diff
@@ -55,6 +55,7 @@ Open Source Components
 Known Issues
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+- Desktop App Windows 10: The taskbar does not flash on receipt of a new message when the setting is enabled `MM-58087 <https://mattermost.atlassian.net/browse/MM-58087>`_.
```
Comment (devinbinnie) [inline source/deploy/desktop-app-changelog.rst:58]: Could we phrase this to say that it may not happen? It only seems to reproduce for certain machines.

**Pull request merged**